src/components/Experience/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import ResumeSection from "../shared/ResumeSection"
+
+import "./styles.css"
+import Work from "./Work"
+
+const Experience = ({ work }) => {
+  const content = work.values.map((each, index) => {
+    return <Work work={each} key={index} />
```

Comment:
nice, you didn't forget the key! I usually always do haha
great using the second parameter on the map function to give they key for each component!